.github/workflows/1.249-lcm.yml

```diff
@@ -0,0 +1,84 @@
+name: Build and test (1.249-lcm, scheduled)
+
+on:
+  schedule:
+    # run every Monday, this refreshes the cache
+    - cron: '13 2 * * 1'
+
+jobs:
+  python-test:
+    name: Python tests
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        test: ["", "-3"]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          ref: '1.249-lcm'
+
+      - name: Run python tests
+        run: bash .github/python-nosetests${{ matrix.test }}.sh
+
+  ocaml-test:
+    name: Ocaml tests
+    runs-on: ubuntu-20.04
+    env:
+      package: "xapi-cli-protocol xapi-client xapi-consts xapi-database xapi-datamodel xapi-types xapi xe"
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          ref: '1.249-lcm'
+
+      - name: Pull configuration from xs-opam
+        run: |
+          curl --fail --silent https://raw.githubusercontent.com/xapi-project/xs-opam/release/stockholm/lcm/tools/xs-opam-ci.env | cut -f2 -d " " > .env
+
+      - name: Load environment file
+        id: dotenv
+        uses: falti/dotenv-action@v0.2.4
+
+      - name: Retrieve date for cache key (year-week)
+        id: cache-key
+        run: echo "::set-output name=date::$(/bin/date -u "+%Y%W")"
+        shell: bash
+
+      - name: Restore opam cache
+        id: opam-cache
+        uses: actions/cache@v2
+        with:
+          path: "~/.opam"
+          # invalidate cache every week, gets built using a scheduled job
+          key: ${{ steps.cache-key.outputs.date }}-1.249
```

Comment:
This cache key should be accessible from the rest of the branches, including `1.249-lcm`, according to documentation.